Docs contributing guide updates (#2768)
dapr/docs: daprdocs/content/en/contributing/docs-contrib/maintainer-guide.md

   ```

1. Navigate to `/README.md`.
1. Update the versions table:

```markdown
| Branch                                                       | Website                    | Description                                                                                      |
| ------------------------------------------------------------ | -------------------------- | ------------------------------------------------------------------------------------------------ |
| [v1.1](https://github.com/dapr/docs) (primary)               | https://docs.dapr.io       | Latest Dapr release documentation. Typo fixes, clarifications, and most documentation goes here. |
| [v1.2](https://github.com/dapr/docs/tree/v1.2) (pre-release) | https://v1-2.docs.dapr.io/ | Pre-release documentation. Doc updates that are only applicable to v1.2+ go here.                |
```

1. In VS Code, search for any `v1.0` references and replace them with `v1.1` as applicable.
1. Commit the staged changes and push to your branch (`release_v1.1`).
1. Open a PR from `release/v1.1` to `v1.1`.
1. Have a docs maintainer or approver review. Wait to merge the PR until release.

### Create new website for future release

> Next, create a new website for the future Dapr release, which will be pointed to from the latest website. To do this, you'll need to:

Reviewer: ```suggestion
Next, create a new website for the future Dapr release, which points to from the latest website. To do this, you'll need to:
```

- Deploy an Azure Static Web App.
- Configure DNS via request from CNCF.

These steps require authentication. 

> ### Create new website for future release

Reviewer: This has the same title as above? This looks wrong. 
Also we need to say in this section that you need MS employee access to create a new Website.

> Next, create a new Azure Static Web App for the future Dapr release, which will be pointed to from the latest website. For this example, we will use v1.2 as the future release.

Reviewer: ```suggestion
Next, create a new Azure Static Web App for the future Dapr release, which points to the latest docs website. For this example, we will use v1.2 as the future release.
```
